views/gitHub-loading.hbs

```diff
@@ -0,0 +1,51 @@
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="ap-local-base-url" content="{{localBaseUrl}}" />
+    <title>{{title}}</title>
+    <link rel="stylesheet" href="/public/css/loading-screen.css" media="all" />
+    <link rel="stylesheet" href="/public/css/ghe-loading.css" media="all" />
+    <link
+      rel="stylesheet"
+      href="/public/aui/aui-prototyping.css"
+      integrity="DTM1Q+8lU7SzJT+FWr0JFisCSZlwfM0GiAKYy7h1s9vIKa/CIh37s9NuOCqIOgK4tmqrjLK4NuWuIPUQNsikHA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <script src="/public/js/jquery.min.js" nonce="{{nonce}}"></script>
+  </head>
+  <body class="gheLoading___body">
+    {{#if isGitHubCloud}}
```

Comment:
Have you thought about how you are going to tie this session screen into this conditional check? Right now we're basically saying, if it's a cloud app, render content 'a', otherwise render content 'b'. Unfortunately, that doesn't really work in the server flow when you take the 'Retrieving data from GitHub Enterprise Server' into account.

When a server customer goes through the flow what screen/s they see will depend upon whether they choose auto/manual app creation.

- auto: when they click create on the auto/manual screen, they'll see the session view with 'Redirecting to your GitHub Enterprise Server instance' before being redirected to GH to enter a name for their app. Once that is done, they'll be redirected back to our app and see 'Retrieving data from GitHub Enterprise Server' while we're saving their app data to the db.
- manual: these customers will only see the second session screen. From the auto/manual screen they'll land on the form. After they click next, they'll see  'Retrieving data from GitHub Enterprise Server' while we're saving their app data to the db.

<img width="648" alt="Screen Shot 2022-07-27 at 3 33 35 pm" src="https://user-images.githubusercontent.com/37155488/181168952-d4f71e4e-fe76-4e2c-8429-8e0b67624ef9.png">

Reply:
Removed the if condition and passed in the `text` from the backend. For the scenario mentioned above, we might have to add some logic to get the desired `text` and then pass that along, and the `gitHub-session.hbs`(loading screen) will simply render it